README.md

```diff
@@ -1,29 +1,151 @@
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[Add class Person as the parent class]**
+- **[Add class Student as a child class of Parent class]**
+- **[Add class Teacher as a child class of Parent class]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+git@github.com:Ibnballo1/oop_school_library.git
+
+Then cd into oop_school_library
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Abdullateef Bello**
+
+- GitHub: [@Ibnballo1](https://github.com/Ibnballo1/)
+- Twitter: [@webprotekh](https://twitter.com/webprotekh)
+- LinkedIn: [Abdullateef Bello](https://www.linkedin.com/in/abdullateef-bello-1b8006228/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURE -->
+## 👥 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- Add new students or teachers.
+- Add new books.
+- Save records of who borrowed a given book and when.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Ibnballo1/oop_school_library/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project, give a ⭐️!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
 
-## Show your support
-Give a ⭐️ if you like this project!
-## Acknowledgments 
 Thanks to all Microverse partners for their advice.
-## License
-This project is [MIT](./MIT.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](C:\Users\J T COMPUTER\Microverse-April18\Module4- Ruby  Databases and Interviewing\oop_school_library\MIT.md) licensed.
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add the correct path to your `MIT.md` license file as the current one is not working. You can just update it to this: `[MIT](./MIT.md)`